docs/standard/modern-web-apps-azure-architecture/choose-between-traditional-web-and-single-page-apps.md

```diff
@@ -1,34 +1,33 @@
 ---
 title: Choose between traditional web apps and single page apps
-description: Architect modern web applications with ASP.NET Core and Microsoft Azure
+description: Architect Modern Web Applications with ASP.NET Core and Azure | Choose Between Traditional Web App or SPA
```

Comment:
one sentence, don't include the name of the book

Reply:
done